src/main/java/org/apache/commons/imaging/formats/jpeg/iptc/IptcParser.java

```diff
@@ -202,6 +209,11 @@ public PhotoshopApp13Data parsePhotoshopSegment(final byte[] bytes, final boolea
             // Debug.debug("recordSize", recordSize + " (0x"
             // + Integer.toHexString(recordSize) + ")");
 
+            if( recordNumber==IptcConstants.IPTC_ENVELOPE_RECORD_NUMBER && recordType==ENV_TAG_CODED_CHARACTER_SET ) {
```

Comment:
Nit-picking, but I think the rest of the code base is using `if(etcetcetc) { }`, without spaces after the parentheses. But we can fix that later.

Reply:
Yes, there some others :) For example, there is not space between if and opening paranthes. I fixed all. I pushed the new changes. 

There have been many commits for this change. If it is merged, is it possible to convert it as single commit?